### ISSUE TITLE
nvda.exe icons are awful

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#355]
**Reported by timeless on 2009-07-03 09:17**
w7 rc; NVDA 0.6p3.2

The image http://www.nvda-project.org/chrome/site/images/nvda_100x100white.png looks reasonably good, but for some reason it seems like someone tried very hard to pixelate the icon when they created the icons for NVDA.exe

http://msdn.microsoft.com/en-us/library/ms997636.aspx
http://msdn.microsoft.com/en-us/library/aa511280.aspx
http://www.axialis.com/tutorials/tutorial-vistaicons.html

There seems to be an icon with resolutions up to 48x48, but it's very rough. I'm assuming that there's a vector original, it should be possible to use it to generate better icons.

Also note that as far as w7 is concerned, NVDA.exe doesn't have any icons at all -- If I manually try to change the icon of NVDA's shortcut to use NVDA.exe for an icon, it says there aren't any.

**Comment by leonardder** [comment]
@JCSTeh: I guess this will be dealt with as part of the new look and feel of NVDA, similar to #5382?

**Comment by jcsteh** [comment]
@feerrenrut, any thoughts? Do you see this problem?

**Comment by feerrenrut** [comment]
In windows 10 I haven't noticed a problem. The start menu, the task bar, and the system tray all have the icon looking as expected.
I haven't tested windows 7 yet. 

When testing with high contrast mode I dont notice any issue. However when NVDA is pinned in the start menu (thus much larger) AND my DPI is set to 175% I can notice a few rough edges. But I have to look pretty closely to notice this. I would say its acceptable even in this case.

@Qchristensen What are your thoughts on this, have you noticed any icon issues?

**Comment by feerrenrut** [comment]
A screen grab of the image listed above:
![image](https://user-images.githubusercontent.com/3491346/27064282-274f2f98-5029-11e7-9b38-2b4d66f9ecbd.png)

Screen grabs of the the ico file at various resolutions. Notice that there is a fair bit of aliasing on the diagonal lines, this could probably be addressed. I think we could export this with a higher maximum resolution support as well.

48x48: 
![image](https://user-images.githubusercontent.com/3491346/27064222-dbaa5cca-5028-11e7-81c1-63b0c1e2b42f.png)
32x32:
![image](https://user-images.githubusercontent.com/3491346/27064232-ebc31a34-5028-11e7-9dbc-35e3a23c4bb3.png)
24x24:
![image](https://user-images.githubusercontent.com/3491346/27064246-fb935370-5028-11e7-98f1-89755bf59caf.png)
16x16:
![image](https://user-images.githubusercontent.com/3491346/27064253-08b862d4-5029-11e7-8dc3-4e8a53c8c8a6.png)

**Comment by feerrenrut** [comment]
We are considering updating the icons for NVDA. We dont think that any of the current issues require an immediate fix, and when new icons are introduced we will try to ensure that none of these issues persist.

Closing as won't fix.
